LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -37,18 +39,15 @@ + (nullable UIWindow *)currentWindow;
 @implementation UIApplication (window)
 
 + (UIWindow *)currentWindow {
-    UIWindow *window;
-#ifdef LTH_IS_APP_EXTENSION
-    window = [UIApplication sharedApplication].keyWindow;
-#else
-    window = [UIApplication sharedApplication].windows.firstObject;
-#endif
-    return window;
+    return [UIApplication sharedApplication].windows.firstObject;
 }
 
 @end
 
+#endif
+
 @interface LTHPasscodeViewController () <UITextFieldDelegate>
+@property (nonatomic, weak)   UIView      *presentingView;
```

Comment:
Do we need to make this `presentingView` only available for `LTH_IS_APP_EXTENSION`? so for non-extensions, we always use key window